src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -360,7 +359,7 @@ class BaseOptionsSelector extends Component {
                                 text={defaultConfirmButtonText}
                                 onPress={this.props.onConfirmSelection}
                                 pressOnEnter
-                                enterKeyEventListenerPriority={2}
+                                enterKeyEventListenerPriority={1}
```

Comment:
We lowered this from 2 to 1 because now the priority for the enter key is 0 correct?

Reply:
I'm guessing that is the case. Do you think we should just leave as 2?

This change came from this PR: https://github.com/Expensify/App/pull/18578